ludwig/encoders/binary_encoders.py

```diff
@@ -35,15 +35,15 @@ def __init__(self, encoder_config=None, **kwargs):
 
         logger.debug(f" {self.name}")
 
-    def forward(self, inputs: torch.Tensor) -> torch.Tensor:
+    def forward(self, inputs: torch.Tensor) -> dict:
```

Comment:
Are there other encoders that aren't forwarding Dict outputs?

Reply:
Can you make this `Dict[str, torch.Tensor]`?

Reply:
@justinxzhao both the `BinaryPassthroughEncoder` and the generic `DenseEncoder` which are the two available encoders for binary features return dictionaries.